src/arc.c

```diff
@@ -27,11 +27,11 @@ char *format_bits(t_dip *dip, int base, int page_id) {
     // Parse bits first
     while (token = strtok(token, ",")) {
         char c = atoi(token) + (char)base;
-        if (c > 31) {
-            printf("error while parsing dip switch: required bit position exceeds 31.\n");
+        if (c > 61) {
+            printf("error while parsing dip switch: required bit position exceeds 61.\n");
```

Comment:
Maybe change this even to
```printf("error while parsing \"%s\" dip switch: required bit position exceeds 61.\n", dip->name);```
so we see where the error happens. Like in line 41.

Reply:
Let's hope nobody needs more than 61 switches :)